rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/ip/DiscoverHardcodedIPAddressRuleProvider.java

```diff
@@ -87,7 +91,8 @@ public void perform(GraphRewrite event, EvaluationContext context, FileLocationM
                     hintBody.append("\n\n");
                     hintBody.append("When migrating environments, hard-coded IP addresses may need to be modified or eliminated.");
                     location.setHint(hintBody.toString());
-
+                    //location.setIssueCategory(IssueCategoryRegistry.loadFromGraph(context, IssueCategoryRegistry.MANDATORY));
+                    location.setIssueCategory(IssueCategoryRegistry.loadFromGraph(context, "cloud-mandatory");
```

Comment:
Please add an entry to the issuecategoryregistry instead of just using a string literal here.